wallet/src/wallet-api/routes.rs

```diff
@@ -851,7 +861,7 @@ async fn buildsponsor(
     let erc20_code: Address = bindings[":erc20"].value.as_string()?.parse()?;
     let sponsor_address: Address = bindings
         .get(":sponsor")
-        .unwrap()
+        .expect("buildsponsor request :sponsor parameter")
```

Comment:
Nit: Should be "requests". Or use "must have" to be consistent with other error messages?